src/ctelnet.cpp

```diff
@@ -2374,7 +2379,8 @@ void cTelnet::postMessage(QString msg)
             QString prefix = body.at(0).left(prefixLength).toUpper();
             QString firstLineTail = body.at(0).mid(prefixLength);
             body.removeFirst();
-            if (prefix.contains(tr("ERROR", "Keep the capisalisation, the translated text at 7 letters max so it aligns nicely")) || prefix.contains(QLatin1String("ERROR"))) {
+            //: Keep the capisalisation, the translated text at 7 letters max so it aligns nicely
```

Comment:
Given that you are now editing this text the Spelling check bot would start moaning about this (repeated) "smelling error" someone (possibly me) made - if it was still working!
```suggestion
            //: Keep the capitalisation, the translated text at 7 letters max so it aligns nicely
```

Reply:
Didn't think of this, thanks!